src/python/gudhi/cover_complex.py

```diff
@@ -23,6 +23,46 @@
 
 from . import SimplexTree, CoverComplex
 
+def _save_to_html(dat, lens, color, param, nums, points, edges, html_output_filename):
+
+    from ._kepler_mapper import KeplerMapper
+    network = {}
+    mapper = KeplerMapper(verbose=0)
+    data = np.zeros((3,3))
+    projected_data = mapper.fit_transform( data, projection="sum", scaler=None )
+
+    num_nodes = nums[0]
+    num_edges = nums[1]
```

Comment:
Would it help the user if `num_nodes` was equal to `len(points)` and `num_edges` was equal to `len(edges)` ?